src/Views/SuccessView.vala

```diff
@@ -43,15 +43,19 @@ public class Sideload.SuccessView : AbstractView {
                 primary_label.label = _("The app was installed successfully");
             }
 
-            secondary_label.label = _("Open it any time from the Applications Menu. Visit %s for app information, updates, and to uninstall.").printf (appstore_name);
+            secondary_label.label = _("Open it any time from the Applications Menu. Visit %s for app information, updates, and to uninstall. Permissions can be changed in <a href='%s'>%s → %s…</a>").printf (
+                appstore_name, "settings://applications/permissions", _("System Settings"), _("Applications")
```

Comment:
Good call pulling out the URL so it doesn't get localized. I wonder if we need a translators comment for context on "Applications" or not?

Reply:
Something like this maybe?

`
/// TRANSLATORS: "System Settings" is related to the title of https://github.com/elementary/switchboard, "Applications" is related to the title of https://github.com/elementary/switchboard-plug-applications
`

Reply:
Yeah that seems pretty comprehensive, let's do that